### PR TITLE
consistency fixes

### PR DESCRIPTION
**Pull request by meltinglava**
```
Duplicates: FIR: FAJO, has duplicate points: -20.0|-10.0.
Duplicates: FIR: FNAN, has duplicate points: -4.166667|6.583333.
Duplicates: FIR: LCCC, has duplicate points: 33.5|30.0.
Duplicates: FIR: LGGG, has duplicate points: 41.706222|26.362361.
Duplicates: FIR: LGGG-E, has duplicate points: 37.848667|24.302.
Duplicates: FIR: LTAA, has duplicate points: 42.116667|29.0.
Duplicates: FIR: LTBB, has duplicate points: 42.116667|29.0.
Duplicates: FIR: TEH-1N, has duplicate points: 38.632414|44.299892.
Duplicates: FIR: OIIX, has duplicate points: 34.317683|60.683031.
Duplicates: FIR: ZBPE, has duplicate points: 45.388056|115.354167.
Duplicates: FIR: ZBAA, has duplicate points: 45.388056|115.354167.
Wrong min/max for sector: ZBAA: stated minimum latitude: 36.316667, actual: 32.125, stated minimum longitude: 110.6, actual: 105.229167, stated maximum latitude: 42.25, actual: 45.434694.
Duplicates: FIR: ZBHH, has duplicate points: 38.733333|109.683333.
Duplicates: FIR: ZBLA, has duplicate points: 46.615278|119.7675.
Duplicates: FIR: ZGZU, has duplicate points: 29.516667|109.4.
Wrong min/max for sector: ZGZU: stated minimum latitude: 17.216667, actual: 20.5, stated maximum latitude: 29.516667, actual: 33.533333.
Wrong min/max for sector: ZSHA-AR22: stated minimum latitude: 21.57, actual: 25.091667, stated minimum longitude: 110.55, actual: 113.95, stated maximum latitude: 26.05, actual: 26.855, stated maximum longitude: 117.5, actual: 116.032222.
Duplicates: FIR: ZGGG, has duplicate points: 29.516667|109.4, 25.091667|115.416667, 26.583333|109.316667.
Wrong min/max for sector: ZGGG: stated minimum latitude: 21.57, actual: 20.5, stated minimum longitude: 110.55, actual: 107.65, stated maximum latitude: 26.05, actual: 33.533333.
Wrong min/max for sector: ZGHA: stated minimum latitude: 25.276667, actual: 25.279167.
Wrong min/max for sector: ZGNN-AR08: stated maximum longitude: 111.34, actual: 111.341667.
Duplicates: FIR: ZGNN, has duplicate points: 23.195833|105.541667.
Wrong min/max for sector: ZGNN: stated maximum latitude: 25.7, actual: 26.583333, stated maximum longitude: 111.306667, actual: 111.341667.
Wrong min/max for sector: ZGZJ: stated minimum longitude: 110.005, actual: 108.8.
Duplicates: FIR: ZJSY, has duplicate points: 19.5|111.5.
Wrong min/max for sector: ZJSY: stated minimum latitude: 17.666667, actual: 14.5, stated maximum latitude: 20.5, actual: 22.505, stated maximum longitude: 111.5, actual: 114.0.
Wrong min/max for sector: ZJSA-OCA: stated minimum latitude: 17.666667, actual: 14.5, stated minimum longitude: 107.189722, actual: 108.433333, stated maximum latitude: 20.5, actual: 19.5, stated maximum longitude: 111.5, actual: 114.0.
Duplicates: FIR: ZJSA, has duplicate points: 19.5|111.5.
Wrong min/max for sector: ZJSA: stated minimum latitude: 20.5, actual: 14.5, stated minimum longitude: 108.0, actual: 107.189722, stated maximum latitude: 14.5, actual: 22.505.
Wrong min/max for sector: ZLLL: stated maximum longitude: 105.05, actual: 109.042778.
Wrong min/max for sector: ZLXY-H: stated minimum latitude: 31.9, actual: 32.45, stated maximum longitude: 110.733333, actual: 109.042778.
Wrong min/max for sector: ZLXY: stated minimum latitude: 32.233333, actual: 31.9, stated minimum longitude: 105.816667, actual: 99.083333, stated maximum latitude: 39.113333, actual: 41.7475, stated maximum longitude: 107.4, actual: 110.866667.
Duplicates: FIR: ZULS, has duplicate points: 30.441667|81.625.
Wrong min/max for sector: ZULS: stated minimum latitude: 28.516667, actual: 27.325164.
Wrong min/max for sector: ZUUU: stated minimum latitude: 26.886667, actual: 24.65.
Wrong min/max for sector: ZSSS: stated minimum latitude: 26.485556, actual: 23.0, stated minimum longitude: 117.278333, actual: 113.95.
Duplicates: FIR: ZSHA, has duplicate points: 36.625556|124.0.
Wrong min/max for sector: ZSAM: stated minimum longitude: 115.408333, actual: 115.016667, stated maximum longitude: 121.448889, actual: 121.473435.
Duplicates: FIR: ZSQD, has duplicate points: 36.625556|124.0.
Duplicates: FIR: ZWWW, has duplicate points: 42.216667|80.25.
Wrong min/max for sector: ZWWW: stated minimum latitude: 38.143333, actual: 30.05, stated minimum longitude: 79.833, actual: 73.6, stated maximum latitude: 49.166667, actual: 49.183333.
Wrong min/max for sector: ZWAK: stated minimum latitude: 30.441667, actual: 30.05.
Wrong min/max for sector: ZYTX: stated maximum latitude: 46.768597, actual: 53.55, stated maximum longitude: 131.31666, actual: 134.76666.
Wrong min/max for sector: ZYSH: stated minimum latitude: 38.0, actual: 36.623889.
Wrong min/max for sector: ZYTL: stated minimum latitude: 38.0, actual: 36.623889.
Duplicates: FIR: PAZA, has duplicate points: 90.0|-168.973056.
Duplicates: FIR: PAZA, has duplicate points: 54.816667|170.2, 54.0|169.0.
Duplicates: FIR: MMEX, has duplicate points: 23.666667|-102.8333.
Duplicates: FIR: MMID, has duplicate points: 17.810556|-90.978333.
Duplicates: FIR: MMTY, has duplicate points: 31.333333|-108.216667.
Duplicates: FIR: MMZT, has duplicate points: 30.75|-120.833333.
Wrong min/max for sector: MMZT: stated minimum latitude: 16.183333, actual: 18.223889, stated minimum longitude: -120.83333, actual: -120.833333, stated maximum latitude: 32.716667, actual: 32.718611.
Duplicates: FIR: MMFO, has duplicate points: 30.0|-120.0.
Duplicates: FIR: UBBA-S, has duplicate points: 39.714169|44.769085, 38.85|46.15.
Duplicates: FIR: UKFV, has duplicate points: 47.108333|35.916667.
Duplicates: FIR: URRV, has duplicate points: 49.983333|41.416667.
Extention not after FIR: The following FIRs has atleast one extention that is not just after it in the file: GMMM, KZAK, LECM, LIMM, NFFF, NFFN, NZZO, SKED, VVHM, WBFC, UHMM, UHPP.
```

### CODE REVIEW

**Comment by NelisV** [comment]
This will be re-run tomorrow once @Adrian2k merges #460.

**Review by Adrian2k — APPROVED** [review]
Waiting for @NelisV to check something before we merge #460.
I will approve this in the meanwhile.

**Review by NelisV — APPROVED** [review]

**Pull request merged**
